cli/pkg/cmdutil/check.go

```diff
@@ -20,23 +26,30 @@ func CheckChain(chain ...PreRunCheck) PreRunCheck {
 	}
 }
 
+// CheckAuth checks if the user is authenticated.
 func CheckAuth(ch *Helper) PreRunCheck {
 	return func(cmd *cobra.Command, args []string) error {
 		// This will just check if token is present in the config
-		if ch.IsAuthenticated() {
-			return nil
+		if !ch.IsAuthenticated() {
+			return fmt.Errorf("command '%s': %w", cmd.Name(), ErrNotAuthenticated)
 		}
-
-		return fmt.Errorf("not authenticated, please run 'rill login'")
+		return nil
```

Comment:
I guess I still don't understand why this doesn't need to check for the `--local` flag because this is what e.g. the `rill project refresh` command looks like:
```
	refreshCmd := &cobra.Command{
		...
		PersistentPreRunE: cmdutil.CheckChain(cmdutil.CheckAuth(ch), cmdutil.CheckOrganization(ch)),
```

Is something else broken somewhere? Because it looks like `cmdutil.CheckAuth` should bounce here for a user who isn't logged in.